portal/pages/api/fileUpload.js

```diff
@@ -1,59 +1,122 @@
 import axios from "axios";
 import { getSession, session } from "next-auth/client";
-import parser from "xml2json";
-
-const Minio = require('minio')
+import parser from "xml-js";
+import { signIn } from "next-auth/client";
+const Minio = require("minio");
 
 const handler = async (req, res) => {
   const session = await getSession({ req });
   if (session) {
+    let requestOptions = {
+      method: "POST",
+      redirect: "follow",
+    };
+
+    const options = {
+      headers: { Authorization: process.env.FUSIONAUTH_API_KEY },
+    };
+    const responseUser = await axios.post(
+      `${process.env.FUSIONAUTH_DOMAIN}/api/login`,
+      {
+        loginId: process.env.UPLOAD_LOGIN_ID,
+        password: process.env.UPLOAD_PASSWORD,
+        applicationId: process.env.UPLOAD_APPLICATION_ID,
+      },
+      options
+    );
+
+    if (!responseUser.data || !responseUser.data.token) {
+      res.status(200).json({ errors: "Error File ", success: null });
```

Comment:
Can we change the error message to something more readable? "File upload: authorisation error"